docs/guides/universal-receiver-delegate/accept-reject-assets.md

```diff
@@ -36,12 +36,12 @@ pragma solidity ^0.8.0;
 
 // This code is only used for guides puprose, it is working but not verified nor audited.
 
-// interfaces
-import {LSP1UniversalReceiverDelegateUP} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP/LSP1UniversalReceiverDelegateUP.sol";
+// modules
+import {LSP1UniversalReceiverDelegateUP} from "@lukso/lsp-smart-contracts/contracts/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP/LSP1UniversalReceiverDelegateUP.sol";
 
 // constants
-import {_TYPEID_LSP7_TOKENSRECIPIENT} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP7DigitalAsset/LSP7Constants.sol";
-import {_TYPEID_LSP8_TOKENSRECIPIENT} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol";
+import {_TYPEID_LSP7_TOKENSRECIPIENT} from "@lukso/lsp-smart-contracts/contracts/LSP7DigitalAsset/LSP7Constants.sol";
```

Comment:
ahah wow, how did we get there. Nice catch

Reply:
These imports are meant to be working from Remix IDE, if they work as `@` then good lets apply this change. If not lets revert

Reply:
Yes, I tested it like that and it works

Reply:
Yes I think it works in Remix with the `@`. Nice catch indeed 👌

Reply:
Ahhh Remix it is ;)